openshift/openshift_transport.go

```diff
@@ -66,7 +66,8 @@ func ParseReference(ref string) (types.ImageReference, error) {
 func NewReference(dockerRef reference.NamedTagged) (types.ImageReference, error) {
 	r := strings.SplitN(dockerRef.RemoteName(), "/", 3)
 	if len(r) != 2 {
-		return nil, errors.Errorf("invalid image reference %s", dockerRef.String())
+		return nil, errors.Errorf("invalid image reference: %s, remote name should be of the form: `registry.example.com/ns/stream:tag`",
+			dockerRef.String())
```

Comment:
`s/remote name should be of the form/expected format/` perhaps? Most importantly please drop the “remote name” concept; the `.RemoteName` method is an internal implementation detail of `docker/reference`, not an user-visible concept.

Also consider using ordinary quotes instead of backquotes, that’s a Go-programmer’s jargon.